server/stats.go

```diff
@@ -131,6 +131,56 @@ func (s *Server) GetStatValue(ctx context.Context, in *pb.GetStatValueRequest) (
 	return &pb.GetStatValueResponse{Value: result}, nil
 }
 
+// GetStatSeries implements API for Mixer.GetStatSeries.
+func (s *Server) GetStatSeries(ctx context.Context, in *pb.GetStatSeriesRequest) (
```

Comment:
I would have expected this to follow exactly the GetStatValue() logic, and then in the final part just return the series instead of a particular date. But this seems to follow the GetStats() flow. I'm wondering why...